Unit-01/02-flask-routing/app.py

```diff
@@ -0,0 +1,24 @@
+from flask import Flask
+
+app= Flask(__name__)
+
+@app.route("/math/<operation>/<num1>/<num2>")
+def do_math(operation, num1, num2):
+	if operation == "addition":
+		output = int(num1) + int(num2)
+	elif operation == "subtraction":
```

Comment:
This is good; However, another way to do this is using a dictionary, just like you can in JavaScript using objects.

```python
def add(a, b):
    return a + b
def subtract(a, b):
    return a - b
def multiply(a, b):
    return a * b
def divide(a / b):
    return a / b

operations = {
    "add": add,
    "subtract": subtract,
    "multiply": multiply,
    "divide": divide
}

# @route handler
def math(operation, num1, num2):
    if operation in operations:
        return f"{operations[operation](num1, num2)}"
```

This is a more declarative style, where you can declare your functions and data structures at the time and simplify the business logic part (i.e. the routing, in this case)

Reply:
Thanks! This is really helpful for understanding declarative style. :)